source/CardGames.py

```diff
@@ -143,3 +168,53 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+  
+
+def question():
+  x = input('It is your turn who whould you like to ask? ')
+  y = input('what card would you like to ask that player for?')
+  if x == 'p1':
+    print('pass the computer to p1')
+    print('\n\n\n\n\n\n\n\n\n\n\n\n')
```

Comment:
On line 178, 189, 200, and 211 instead of printing a new line and repeating it, you could do print('\n"*12). It will still print the same amount of lines, but the code will look more clean.